demos/kqueen_azure-demo.sh

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env bash
+
+################################################
+# include the magic
+################################################
+test -f ./demo-magic.sh || curl --silent https://raw.githubusercontent.com/paxtonhare/demo-magic/master/demo-magic.sh > demo-magic.sh
+. ./demo-magic.sh -n
+
+################################################
+# Configure the options
+################################################
+
+#
+# speed at which to simulate typing. bigger num = faster
+#
+TYPE_SPEED=40
+
+# Uncomment to run non-interactively
+export PROMPT_TIMEOUT=0
+
+# No wait
+export NO_WAIT=true
+
+#
+# custom prompt
+#
+# see http://www.tldp.org/HOWTO/Bash-Prompt-HOWTO/bash-prompt-escape-sequences.html for escape sequences
+#
+DEMO_PROMPT="${GREEN}➜ ${CYAN}$ "
+
+docker-compose -f ../docker-compose.yml -f ../docker-compose.demo.yml stop
+docker-compose -f ../docker-compose.yml -f ../docker-compose.demo.yml rm -f
+
+# hide the evidence
+clear
+
+p  "### KQueen demo with Azure Provisioner"
+wait
+
+p  '#'
+p  '# Create a new Resource Group:'
+wait
+pe 'RESOURCE_GROUP_NAME="kqueen-demo-rg"'
+pe 'az group create --name "$RESOURCE_GROUP_NAME" --location westeurope'
+
+p  '#'
+p  '# Create a service principal that will store permissions to manage resources in the a specified subscription:'
+wait
```

Comment:
seems need to provide description to user, `time to set own creds`

Reply:
I do not understand what you mean... What should I fix exactly?

Reply:
how do user should define its credentials there for demo?

Reply:
Azure authentication is not part of the script. That's part of the "az" command and it's settings. 
This is more KQueen demo than the Azure demo. 

The login/password for the service principal are defined here:

```
pe 'SECRET="my_secret_password"'
pe 'SERVICE_PRINCIPAL_NAME="kqueen-demo-sp"'
```

Reply:
okok, if so, maybe need to add smth like:
> +p  '# Define your Azure credentials here?'

Reply:
I replaced `SECRET` variable by `SERVICE_PRINCIPAL_PASSWORD` to have it more clear:

```
pe 'SERVICE_PRINCIPAL_PASSWORD="my_secret_password"'
pe 'SERVICE_PRINCIPAL_NAME="kqueen-demo-sp"'
pe 'az ad sp create-for-rbac --role="Contributor" --scopes="/subscriptions/$SUBSCRIPTION_ID" --name "$SERVICE_PRINCIPAL_NAME" --password "$SERVICE_PRINCIPAL_PASSWORD"'
```